index.js

```diff
@@ -106,6 +107,10 @@
       } else if (  ( X.lookup('foam.ui.View').isInstance(obj) )
                    || ( 'CView' in GLOBAL && CView.isInstance(obj) ) ) {
         view = obj;
+      } else if ( X.lookup('foam.u2.Element').isInstance(obj) ) {
+        document.body.insertAdjacentHTML('beforeend', obj.outerHTML);
+        obj.load();
+        return;
       } else if ( obj.toView_ ) {
```

Comment:
Element doesn't have toE, or at least it shouldn't.  toE is to adapt something that isn't an Element to an Element.

foam.graphics.CView will probably get a toE soon.